sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_common/_base_eval.py

```diff
@@ -70,6 +72,13 @@ class EvaluatorBase(ABC, Generic[T_EvalValue]):
     :type not_singleton_inputs: List[str]
     :param eval_last_turn: If True, only the last turn of the conversation will be evaluated. Default is False.
     :type eval_last_turn: bool
+    :param conversation_aggregation_type: The type of aggregation to perform on the per-turn results of a conversation
+        to produce a single result.
+        Default is ~azure.ai.evaluation.AggregationType.MEAN.
+    :type conversation_aggregation_type: ~azure.ai.evaluation.AggregationType
+    :param conversation_aggregator_override: A function that will be used to aggregate per-turn results. If provided,
```

Comment:
With `conversation_aggregator_override`, do we still need AggreationType.Custom?

Reply:
I think so. The Custom enum is only for readout purposes, it's not a valid input. It's just there for when you want to convert a function back to the enum to see what you're working with (ex:  you want to print out what aggregation method all your evals are using).

Reply:
I see.